### PR TITLE
Add test that verifies a load balancer's IP doesn't change when its port

### PR DESCRIPTION
**Pull request by a-robinson**
is changed.

Tests part of #13333

### CODE REVIEW

File: test/e2e/service.go [hunk 1]
```diff
@@ -548,6 +548,16 @@ var _ = Describe("Services", func() {
 			i++
 		}
 
+		By("updating service's port " + serviceName + " and reaching it at the same IP")
+		service, err = updateService(f.Client, f.Namespace.Name, serviceName, func(s *api.Service) {
+			s.Spec.Ports[0].Port = 19482
```
Comment (ixdy) [inline test/e2e/service.go]: where does 19482 come from?

Comment (a-robinson) [inline test/e2e/service.go]: It's entirely arbitrary. Is there a different port you'd recommend instead?

Comment (ixdy) [inline test/e2e/service.go]: where did the original port come from? do we have some guarantee this port is different from the original?

Comment (a-robinson) [inline test/e2e/service.go]: The original port used is 80, and is hard-coded all over the place throughout this file. It originally comes from the BuildServiceSpec function.

I was originally going to make the port here the original port+1, but using a reserved port for something it's not intended for felt dirty

**Comment by a-robinson** [comment]
@ixdy - If there are any changes you want with this, could you let me know today? I'll be offline all next week.

**Pull request merged**